### PR TITLE
Minor cleanup to build files

### PR DESCRIPTION
**Pull request by FlorianMichael**
These are mainly changes I do every time I set up a new fabric mod using the template, so I figured I could just pull request/suggest them.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -33,17 +32,14 @@ loom {
 dependencies {
 	// To change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
-	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
+	mappings "net.fabricmc:yarn:${project.yarn_version}:v2"
```
Comment (Linguardium) [inline build.gradle:35]: This no longer matches the develop page for versioning 

https://fabricmc.net/develop/

**Comment by Linguardium** [comment]
Most of these changes seen unnecessary, is there logical reasons that these changes are made?

File: build.gradle [hunk 2]
```diff
@@ -1,13 +1,12 @@
 plugins {
-	id 'fabric-loom' version '1.8-SNAPSHOT'
-	id 'maven-publish'
+	id "fabric-loom" version "1.8-SNAPSHOT"
+	id "maven-publish"
```
Comment (Earthcomputer) [inline build.gradle:3]: I don't mean to start a war or anything, but I think single quotes are better than double quotes where possible in Groovy. Double quotes create a `GString` (which is coerced to a `String` almost everywhere), whereas single quotes directly create a `String`. IMO you should only use double quotes when formatting (with `$`) is used.

File: build.gradle (hunk 1 above)
Comment (Earthcomputer) [reply build.gradle:35]: Yeah they'll have to be updated there too if this is merged

**Comment by modmuss50** [comment]
Thanks for the PR, this repo is generated from the template generator on the website: https://github.com/FabricMC/fabricmc.net/tree/main/scripts/src/lib/template

**Pull request closed without merging**